### PR TITLE
UI rework Export to Mobile

### PR DESCRIPTION
**Pull request by khalilbenihoud**
I gave a bit of UI Polish to the **Export to mobile** feature.

I updated the:
- QR code size (I did a scan and worked perfectly with this size)
- No more scroll inside the modal
- We previsouly used the Send icon, I changed it to the QR code to match the one on Mobile.
- Updated some css and alignments.
- Updated some icons (Hide, Ban, Chevrons..)

Screenshot 2020-08-27 at 10 57 26

Screenshot 2020-08-27 at 10 57 30

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (valpinkman) [inline .vscode/settings.json:1]: do not commit this file

File: src/renderer/icons/Ban.js [hunk 2]
```diff
@@ -3,9 +3,11 @@
...
+      fill-rule="evenodd"
```
Comment (valpinkman) [inline src/renderer/icons/Ban.js:8]: fillRule

File: src/renderer/icons/Ban.js [hunk 3]
```diff
@@ -3,9 +3,11 @@
...
+      clip-rule="evenodd"
```
Comment (valpinkman) [inline src/renderer/icons/Ban.js:9]: clipRule

File: src/renderer/icons/Atom.js [hunk 4]
```diff
@@ -2,16 +2,12 @@
...
-  />
-);
-
 const Atom = ({ size }: { size: number }) => (
   <svg viewBox="0 0 16 16" height={size} width={size}>
-    {path}
+    <path
```
Comment (valpinkman) [inline src/renderer/icons/Atom.js:7]: those were exported to `const path` for performance

File: src/renderer/screens/accounts/OptionsButton.js [hunk 5]
```diff
@@ -14,6 +14,7 @@ import Tooltip from "~/renderer/components/Tooltip";
 import IconDots from "~/renderer/icons/Dots";
 import IconDownloadCloud from "~/renderer/icons/DownloadCloud";
 import IconSend from "~/renderer/icons/Send";
```
Comment (valpinkman) [inline src/renderer/screens/accounts/OptionsButton.js:16]: remove unused import

**Review by valpinkman — CHANGES_REQUESTED** [review]
needs to be rebased also

**Comment by gre** [comment]
```diff
diff --git a/.vscode/settings.json b/.vscode/settings.json
new file mode 100644
index 000000000..3b6641073
--- /dev/null
+++ b/.vscode/settings.json
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
\ No newline at end of file
diff --git a/src/renderer/components/AssetDistribution/Row.js b/src/renderer/components/AssetDistribution/Row.js
index 4ba56e3c5..cf0ec11d2 100644
--- a/src/renderer/components/AssetDistribution/Row.js
+++ b/src/renderer/components/AssetDistribution/Row.js
@@ -118,7 +118,7 @@ const Row = ({ item: { currency, amount, distribution }, isVisible }: Props) =>
       <Distribution>
         {!!distribution && (
           <>
-            <Text ff="Inter" color="palette.text.shade100" fontSize={3}>
+            <Text ff="Inter|Medium" color="palette.text.shade100" fontSize={3}>
               {`${percentage}%`}
             </Text>
             <Bar progress={isVisible ? percentage : "0"} progressColor={color} />
diff --git a/src/renderer/components/Exporter/ExportInstructions.js b/src/renderer/components/Exporter/ExportInstructions.js
index 743d66b34..2009f9de6 100644
--- a/src/renderer/components/Exporter/ExportInstructions.js
+++ b/src/renderer/components/Exporter/ExportInstructions.js
@@ -7,13 +7,13 @@ import Text from "~/renderer/components/Text";
 import BulletRow from "~/renderer/components/BulletRow";

 const BulletRowIcon = styled(Box).attrs(() => ({
-  ff: "Inter|Regular",
+  ff: "Inter|SemiBold",
   fontSize: 10,
   textAlign: "center",
   color: "wallet",
   pl: 2,
 }))`
-  background-color: rgba(100, 144, 241, 0.2);
+  background-color: ${p => p.theme.colors.blueTransparentBackground};
   border-radius: 12px;
   display: inline-flex;
   height: 18px;
@@ -30,10 +30,10 @@ const Plus = styled(Box).attrs(() => ({
   pl: 2,
 }))`
   display: inline-flex;
-  background-color: rgba(100, 144, 241, 0.2);
+  background-color: ${p => p.theme.colors.blueTransparentBackground};
   border-radius: 4px;
   height: 14px;
-  line-height: 14px;
+  line-height: 12px;
   width: 14px;
   padding: 0;
 `;
@@ -90,12 +90,12 @@ const stepsImportMobile = [

 const ExportInstructions = () => (
   <>
-    <Box shrink style={{ width: 330, fontSize: 13, marginTop: 20 }}>
+    <Box shrink style={{ fontSize: 13, margin: 15 }}>
       <Text ff="Inter|SemiBold" color="palette.text.shade100">
         <Trans i18nKey="settings.export.modal.listTitle" />
       </Text>
     </Box>
-    <Box style={{ width: 330 }}>
+    <Box>
       {stepsImportMobile.map(step => (
         <BulletRow key={step.key} step={step} />
       ))}
diff --git a/src/renderer/components/Exporter/index.js b/src/renderer/components/Exporter/index.js
index e1f255b4c..2d1b83c84 100644
--- a/src/renderer/components/Exporter/index.js
+++ b/src/renderer/components/Exporter/index.js
@@ -17,7 +17,7 @@ type Props = OwnProps & {
 const Exporter = ({ accounts }: Props) => (
   <Box justifyContent="center" alignItems="center">
     <Box flow={2}>
-      <QRCodeExporter accounts={accounts} size={330} />
+      <QRCodeExporter accounts={accounts} size={208} />
     </Box>
     <ExportInstructions />
   </Box>
diff --git a/src/renderer/components/Price.js b/src/renderer/components/Price.js
index 00650a6b0..39e0bf031 100644
--- a/src/renderer/components/Price.js
+++ b/src/renderer/components/Price.js
@@ -41,7 +41,7 @@ type Props = OwnProps & {
 };

 const PriceWrapper = styled(Box).attrs(() => ({
-  ff: "Inter",
+  ff: "Inter|Medium",
   horizontal: true,
 }))`
   line-height: 1.2;
diff --git a/src/renderer/icons/AngleDown.js b/src/renderer/icons/AngleDown.js
index 794aa29ea..684d02a12 100644
--- a/src/renderer/icons/AngleDown.js
+++ b/src/renderer/icons/AngleDown.js
@@ -3,10 +3,10 @@
 import React from "react";

 const AngleDown = ({ size, color = "currentColor" }: { size: number, color?: string }) => (
-  <svg viewBox="0 0 16 14" height={size} width={size * 0.875}>
+  <svg viewBox="0 0 16 16" height={size} width={size * 0.875}>
     <path
       fill={color}
-      d="M7.70123023 10.2169906L3.62390158 6.53574061c-.16520211-.146875-.16520211-.384375 0-.53125l.24956063-.221875c.16520211-.146875.43233744-.146875.59753955 0L8 8.97949061l3.5289982-3.19375c.1652022-.146875.4323375-.146875.5975396 0l.2495606.221875c.1652021.146875.1652021.384375 0 .53125L8.29876977 10.2201156c-.16520211.14375-.43233743.14375-.59753954-.003125z"
+      d="M7.71875 10.875C7.875 11.0312 8.09375 11.0312 8.25 10.875L12.875 6.28125C13.0312 6.15625 13.0312 5.90625 12.875 5.75L12.25 5.15625C12.125 5 11.875 5 11.7188 5.15625L8 8.84375L4.25 5.15625C4.09375 5 3.875 5 3.71875 5.15625L3.09375 5.75C2.9375 5.90625 2.9375 6.15625 3.09375 6.28125L7.71875 10.875Z"
     />
   </svg>
 );
diff --git a/src/renderer/icons/AngleUp.js b/src/renderer/icons/AngleUp.js
index ff2381495..631bb6169 100644
--- a/src/renderer/icons/AngleUp.js
+++ b/src/renderer/icons/AngleUp.js
@@ -2,16 +2,12 @@

 import React from "react";

-const path = (
-  <path
-    fill="currentColor"
-    d="M10.345 6.414l6.012 5.734c.19.184.19.48 0 .665l-.804.773a.502.502 0 0 1-.69 0L10 8.957l-4.862 4.629a.502.502 0 0 1-.69 0l-.805-.774a.457.457 0 0 1 0-.664l6.012-5.734c.19-.184.5-.184.69 0z"
-  />
-);
-
-const AngleUp = ({ size }: { size: number }) => (
-  <svg viewBox="0 0 20 20" height={size} width={size * 0.875}>
-    {path}
+const AngleUp = ({ size, color = "currentColor" }: { size: number, color?: string }) => (
+  <svg viewBox="0 0 16 16" height={size} width={size * 0.875}>
+    <path
+      fill={color}
+      d="M8.25 5.15625C8.09375 5 7.875 5 7.71875 5.15625L3.09375 9.71875C2.9375 9.875 2.9375 10.125 3.09375 10.25L3.71875 10.875C3.875 11.0312 4.09375 11.0312 4.25 10.875L8 7.1875L11.7188 10.875C11.875 11.0312 12.125 11.0312 12.25 10.875L12.875 10.25C13.0312 10.125 13.0312 9.875 12.875 9.71875L8.25 5.15625Z"
+    />
   </svg>
 );

diff --git a/src/renderer/icons/Atom.js b/src/renderer/icons/Atom.js
index 62fd4b88f..39e1e5a47 100644
--- a/src/renderer/icons/Atom.js
+++ b/src/renderer/icons/Atom.js
@@ -2,16 +2,12 @@

 import React from "react";

-const path = (
-  <path
-    fill="currentColor"
-    d="M15.246.803c-.954-.95-2.618-1.02-4.695-.195-.827.33-1.689.81-2.551 1.375C7.138 1.418 6.276.938 5.449.608 3.377-.217 1.71-.15.754.803-.615 2.166-.058 5.007 1.926 8.018-.058 11.028-.615 13.87.754 15.233c.513.51 1.231.767 2.108.767.754 0 1.627-.19 2.587-.572.827-.33 1.689-.81 2.551-1.375.862.565 1.724 1.045 2.551 1.375.96.382 1.832.572 2.588.572.876 0 1.594-.256 2.107-.767 1.369-1.363.812-4.204-1.172-7.215 1.984-3.01 2.541-5.852 1.172-7.215zm-4.268.864c1.597-.636 2.857-.656 3.459-.058.82.816.46 2.934-1.07 5.401a22.166 22.166 0 0 0-2.03-2.315A22.114 22.114 0 0 0 9.01 2.697c.667-.411 1.33-.776 1.969-1.03zm-9.414-.058c.291-.29.737-.434 1.3-.434.601 0 1.335.164 2.159.492.638.254 1.301.618 1.969 1.03a22.186 22.186 0 0 0-2.329 1.998 22.166 22.166 0 0 0-2.03 2.315c-1.53-2.467-1.89-4.585-1.07-5.4zm3.458 12.76c-1.595.636-2.855.657-3.459.057-.82-.815-.46-2.933 1.07-5.4a22.166 22.166 0 0 0 2.03 2.314c.75.747 1.536 1.403 2.328 1.999-.667.411-1.33.776-1.969 1.03zm.45-3.835a20.63 20.63 0 0 1-2.157-2.516A20.87 20.87 0 0 1 8 3.36a20.897 20.897 0 0 1 4.685 4.658A20.87 20.87 0 0 1 8 12.676a20.93 20.93 0 0 1-2.527-2.142zm8.964 3.892c-.602.6-1.862.58-3.459-.057-.638-.254-1.301-.619-1.969-1.03a22.186 22.186 0 0 0 2.329-1.999 22.166 22.166 0 0 0 2.03-2.315c1.53 2.468 1.89 4.586 1.07 5.401zM8 6.022a2.002 2.002 0 0 0-2.005 1.996c0 1.1.9 1.995 2.005 1.995a2.002 2.002 0 0 0 2.005-1.995c0-1.1-.9-1.996-2.005-1.996zm0 2.851a.857.857 0 0 1-.859-.855c0-.473.385-.855.859-.855.475 0 .859.382.859.855A.857.857 0 0 1 8 8.873z"
-  />
-);
-
 const Atom = ({ size }: { size: number }) => (
   <svg viewBox="0 0 16 16" height={size} width={size}>
-    {path}
+    <path
+      fill="currentColor"
+      d="M13.4062 8C15.0938 5.40625 15.5 2.96875 14.25 1.75C13.0312 0.5 10.5938 0.90625 8 2.59375C5.375 0.90625 2.9375 0.5 1.71875 1.75C0.46875 2.96875 0.875 5.40625 2.5625 8C0.875 10.625 0.46875 13.0625 1.71875 14.2812C3.625 16.1875 7.40625 13.8125 8 13.4375C8.5625 13.8125 12.3438 16.1875 14.25 14.2812C15.5 13.0625 15.0938 10.625 13.4062 8ZM3.5 6.6875C2.4375 4.8125 2.25 3.3125 2.75 2.78125C3.46875 2.09375 5.34375 2.78125 6.65625 3.53125C5.6875 4.3125 4.28125 5.71875 3.5 6.6875ZM6.65625 12.4688C4.78125 13.5625 3.28125 13.75 2.75 13.2188C2.25 12.6875 2.4375 11.2188 3.5 9.34375C4.28125 10.3125 5.6875 11.7188 6.65625 12.4688ZM11.5625 8C10.5312 9.4375 9.40625 10.5625 7.96875 11.5938C6.5625 10.5625 5.46875 9.4375 4.40625 8C5.25 6.875 6.84375 5.25 7.96875 4.40625C9.125 5.25 10.7188 6.875 11.5625 8ZM9 8C9 7.46875 8.53125 7 8 7H7.96875C7.4375 7 6.96875 7.46875 6.96875 8C6.96875 8.5625 7.4375 9 7.96875 9C8.53125 9 9 8.5625 9 8ZM12.4688 9.34375C13.5312 11.2188 13.7188 12.6875 13.2188 13.2188C12.6875 13.75 11.1875 13.5625 9.3125 12.4688C10.2812 11.7188 11.6875 10.3125 12.4688 9.34375ZM9.3125 3.53125C10.625 2.78125 12.5 2.09375 13.2188 2.78125C13.7188 3.3125 13.5312 4.8125 12.4688 6.6875C11.6875 5.71875 10.2812 4.3125 9.3125 3.53125Z"
+    />
   </svg>
 );

diff --git a/src/renderer/icons/Ban.js b/src/renderer/icons/Ban.js
index 5004d37fb..2e885f700 100644
--- a/src/renderer/icons/Ban.js
+++ b/src/renderer/icons/Ban.js
@@ -3,9 +3,11 @@
 import React from "react";

 const Ban = ({ size = 16, color = "currentColor" }: { size?: number, color?: string }) => (
-  <svg viewBox="0 0 14 15" height={size} width={size}>
+  <svg height={size} width={size} viewBox="0 0 22 22">
     <path
-      d="M7 0.96875C8.20312 0.96875 9.35156 1.29688 10.3906 1.89844C11.4297 2.5 12.25 3.32031 12.8516 4.35938C13.4531 5.39844 13.7812 6.54688 13.7812 7.75C13.7812 8.98047 13.4531 10.1016 12.8516 11.1406C12.25 12.1797 11.4297 13.0273 10.3906 13.6289C9.35156 14.2305 8.20312 14.5312 7 14.5312C5.76953 14.5312 4.64844 14.2305 3.60938 13.6289C2.57031 13.0273 1.72266 12.1797 1.12109 11.1406C0.519531 10.1016 0.21875 8.98047 0.21875 7.75C0.21875 6.54688 0.519531 5.39844 1.12109 4.35938C1.72266 3.32031 2.57031 2.5 3.60938 1.89844C4.64844 1.29688 5.76953 0.96875 7 0.96875ZM10.5547 4.19531C9.95312 3.62109 9.29688 3.21094 8.53125 2.96484C7.76562 2.74609 7 2.69141 6.23438 2.80078C5.44141 2.91016 4.73047 3.18359 4.10156 3.62109L11.1289 10.6484C11.5664 10.0195 11.8398 9.30859 11.9492 8.51562C12.0586 7.75 12.0039 6.98438 11.7852 6.21875C11.5391 5.45312 11.1289 4.79688 10.5547 4.19531ZM3.44531 11.3047C4.01953 11.9062 4.67578 12.3164 5.44141 12.5352C6.20703 12.7812 6.97266 12.8359 7.76562 12.7266C8.53125 12.6172 9.24219 12.3164 9.89844 11.8789L2.87109 4.85156C2.43359 5.50781 2.13281 6.21875 2.02344 6.98438C1.91406 7.77734 1.96875 8.54297 2.21484 9.30859C2.43359 10.0742 2.84375 10.7305 3.44531 11.3047Z"
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M5.38211 3.96826C6.92181 2.73654 8.87489 2 11 2C15.9706 2 20 6.02944 20 11C20 13.1251 19.2635 15.0782 18.0317 16.6179L5.38211 3.96826ZM3.96794 5.38252C2.73641 6.92216 2 8.87508 2 11C2 15.9706 6.02944 20 11 20C13.1249 20 15.0778 19.2636 16.6175 18.0321L3.96794 5.38252ZM11 0C4.92487 0 0 4.92487 0 11C0 17.0751 4.92487 22 11 22C17.0751 22 22 17.0751 22 11C22 4.92487 17.0751 0 11 0Z"
       fill={color}
     />
   </svg>
diff --git a/src/renderer/icons/EyeSlash.js b/src/renderer/icons/EyeSlash.js
deleted file mode 100644
index 9b68f139f..000000000
--- a/src/renderer/icons/EyeSlash.js
+++ /dev/null
@@ -1,19 +0,0 @@
-// @flow
-
-import React from "react";
-
-const path = (
-  <path
-    fill="currentColor"
-    fillRule="nonzero"
-    d="M8 10.156c.181-.002.472-.031.65-.066l1.297 1.02A7.653 7.653 0 0 1 8 11.374c-3.073 0-5.757-1.818-7.113-4.504A.987.987 0 0 1 .8 6.5c0-.109.039-.275.087-.371.256-.508.573-.973.918-1.412l2.622 2.058c.142 1.89 1.677 3.38 3.573 3.38zm7.845 1.475a.462.462 0 0 1 .155.321.478.478 0 0 1-.084.25l-.491.641a.45.45 0 0 1-.316.157.462.462 0 0 1-.246-.086L.154 1.37A.462.462 0 0 1 0 1.048C0 .97.038.859.084.798l.49-.641A.45.45 0 0 1 .892 0c.076 0 .186.038.245.086l3.18 2.496c.951-.527 2.601-.956 3.684-.957 3.073 0 5.757 1.818 7.113 4.504a.987.987 0 0 1 .087.37.987.987 0 0 1-.087.371c-.42.83-1.33 1.991-2.031 2.592l2.763 2.17zm-4.593-3.605c.216-.465.348-.977.348-1.526v-.002c0-2.017-1.612-3.654-3.598-3.654H8c-.712 0-1.733.377-2.279.84l1.84 1.445c.019-.069.036-.183.039-.254 0-.22-.105-.54-.233-.716.18-.053.477-.097.664-.097 1.308 0 2.37 1.078 2.37 2.406L10.4 6.5a2.922 2.922 0 0 1-.13.755l.982.77z"
-  />
-);
-
-const EyeSlash = ({ width = 16, height = 13 }: { width?: number, height?: number }) => (
-  <svg viewBox="0 0 16 13" height={height} width={width}>
-    {path}
-  </svg>
-);
-
-export default EyeSlash;
diff --git a/src/renderer/icons/Settings.js b/src/renderer/icons/Settings.js
index 86b67e3c5..1a6edce0b 100644
--- a/src/renderer/icons/Settings.js
+++ b/src/renderer/icons/Settings.js
@@ -2,16 +2,12 @@

 import React from "react";

-const path = (
-  <path
-    fill="currentColor"
-    d="M13.94 8.583H14a.583.583 0 0 0 0-1.166h-.116a1.85 1.85 0 0 1-1.693-1.122.75.75 0 0 1-.053-.189 1.854 1.854 0 0 1 .432-1.903l.04-.04a.583.583 0 1 0-.826-.826l-.046.046a1.847 1.847 0 0 1-2.033.373A1.851 1.851 0 0 1 8.583 2.06V2a.583.583 0 0 0-1.166 0v.116A1.85 1.85 0 0 1 6.295 3.81a.75.75 0 0 1-.189.053 1.854 1.854 0 0 1-1.903-.432l-.04-.04a.583.583 0 1 0-.826.826l.046.046c.529.54.675 1.349.388 1.997A1.856 1.856 0 0 1 2.06 7.47H2a.583.583 0 0 0 0 1.167h.116a1.847 1.847 0 0 1 1.69 1.114c.306.691.159 1.5-.376 2.046l-.04.04a.583.583 0 1 0 .826.826l.046-.046c.54-.529 1.349-.675 1.997-.388A1.856 1.856 0 0 1 7.47 13.94V14a.583.583 0 0 0 1.167 0v-.116a1.847 1.847 0 0 1 1.114-1.69 1.853 1.853 0 0 1 2.046.376l.04.04a.583.583 0 1 0 .826-.826l-.046-.046a1.85 1.85 0 0 1-.371-2.037 1.851 1.851 0 0 1 1.694-1.118zm-.33-2.807c.065.087.167.14.277.14H14a2.083 2.083 0 0 1 0 4.167h-.057a.35.35 0 0 0-.32.212.35.35 0 0 0 .06.388l.04.04a2.083 2.083 0 1 1-2.947 2.947c-.136-.134-.289-.162-.427-.1a.349.349 0 0 0-.212.317V14a2.083 2.083 0 0 1-4.167 0c-.003-.186-.094-.313-.273-.38a.348.348 0 0 0-.38.064l-.04.04a2.083 2.083 0 1 1-2.947-2.948c.134-.136.162-.289.1-.427a.349.349 0 0 0-.317-.212H2A2.083 2.083 0 0 1 2 5.97c.186-.003.313-.094.38-.273a.348.348 0 0 0-.064-.38l-.04-.04A2.083 2.083 0 1 1 5.225 2.33c.136.134.289.162.42.104a.75.75 0 0 1 .132-.044.348.348 0 0 0 .14-.277V2a2.083 2.083 0 1 1 4.167 0v.057c0 .14.084.265.22.324.13.057.284.03.38-.065l.04-.04a2.083 2.083 0 1 1 2.947 2.948c-.134.136-.162.289-.104.42a.75.75 0 0 1 .044.132zM8 5.25a2.75 2.75 0 1 0 0 5.5 2.75 2.75 0 0 0 0-5.5zm0 1.5a1.25 1.25 0 1 1 0 2.5 1.25 1.25 0 0 1 0-2.5z"
-  />
-);
-
-const Settings = ({ size }: { size: number }) => (
-  <svg viewBox="0 0 16 16" height={size} width={size}>
-    {path}
+const Settings = ({ size, color = "currentColor" }: { size: number, color?: string }) => (
+  <svg width="16" height="16" viewBox="0 0 16 16">
+    <path
+      d="M14.125 7.90625L15.125 7.34375C15.4062 7.1875 15.5625 6.84375 15.4688 6.5C15.125 5.1875 14.4062 4 13.4688 3.03125C13.2188 2.8125 12.8438 2.75 12.5625 2.90625L11.5625 3.5C11.3125 3.3125 10.8438 3.03125 10.5312 2.90625V1.75C10.5312 1.4375 10.2812 1.09375 9.96875 1.03125C8.6875 0.6875 7.28125 0.6875 6 1.03125C5.65625 1.125 5.4375 1.40625 5.4375 1.75V2.90625C5.125 3.03125 4.65625 3.3125 4.40625 3.5L3.40625 2.90625C3.125 2.75 2.75 2.8125 2.5 3.03125C1.5625 4 0.84375 5.1875 0.5 6.5C0.40625 6.84375 0.5625 7.1875 0.84375 7.34375L1.84375 7.90625C1.8125 8.09375 1.8125 8.34375 1.8125 8.5C1.8125 8.6875 1.8125 8.9375 1.84375 9.09375L0.84375 9.6875C0.5625 9.84375 0.40625 10.1875 0.5 10.5312C0.84375 11.8438 1.5625 13.0312 2.5 14C2.75 14.2188 3.125 14.2812 3.40625 14.125L4.40625 13.5312C4.65625 13.7188 5.125 14 5.4375 14.125V15.2812C5.4375 15.5938 5.6875 15.9375 6 16C7.28125 16.3438 8.6875 16.3438 9.96875 16C10.3125 15.9062 10.5312 15.625 10.5312 15.2812V14.125C10.8438 14 11.3125 13.7188 11.5625 13.5312L12.5625 14.125C12.8438 14.2812 13.2188 14.2188 13.4688 14C14.4062 13.0312 15.125 11.8438 15.4688 10.5312C15.5625 10.1875 15.4062 9.84375 15.125 9.6875L14.125 9.09375C14.1562 8.71875 14.1562 8.3125 14.125 7.90625ZM12.4688 9.90625L13.8438 10.6875C13.6562 11.25 13.1562 12.0625 12.7812 12.5L11.4062 11.7188C10.4062 12.5625 10.2812 12.6562 9.03125 13.0938V14.6875C8.75 14.7188 8.28125 14.7812 8 14.7812C7.6875 14.7812 7.21875 14.7188 6.9375 14.6875V13.0938C5.6875 12.6562 5.53125 12.5625 4.5625 11.7188L3.1875 12.5C2.8125 12.0625 2.3125 11.25 2.125 10.6875L3.5 9.90625C3.25 8.59375 3.25 8.4375 3.5 7.125L2.125 6.34375C2.375 5.6875 2.71875 5.0625 3.1875 4.53125L4.5625 5.3125C5.5625 4.46875 5.6875 4.375 6.9375 3.9375V2.34375C7.21875 2.3125 7.6875 2.25 8 2.25C8.28125 2.25 8.75 2.3125 9.03125 2.34375V3.9375C10.2812 4.375 10.4375 4.46875 11.4062 5.3125L12.7812 4.53125C13.1562 4.96875 13.6562 5.78125 13.8438 6.34375L12.4688 7.125C12.7188 8.4375 12.7188 8.59375 12.4688 9.90625ZM8 5.5C6.34375 5.5 5 6.875 5 8.5C5 10.1562 6.34375 11.5 8 11.5C9.625 11.5 11 10.1562 11 8.5C11 6.875 9.625 5.5 8 5.5ZM8 10C7.15625 10 6.5 9.34375 6.5 8.5C6.5 7.6875 7.15625 7 8 7C8.8125 7 9.5 7.6875 9.5 8.5C9.5 9.34375 8.8125 10 8 10Z"
+      fill={color}
+    />
   </svg>
 );

diff --git a/src/renderer/screens/accounts/OptionsButton.js b/src/renderer/screens/accounts/OptionsButton.js
index 28f89b776..ab3cade0a 100644
--- a/src/renderer/screens/accounts/OptionsButton.js
+++ b/src/renderer/screens/accounts/OptionsButton.js
@@ -13,7 +13,7 @@ import Switch from "~/renderer/components/Switch";
 import Tooltip from "~/renderer/components/Tooltip";
 import IconDots from "~/renderer/icons/Dots";
 import IconDownloadCloud from "~/renderer/icons/DownloadCloud";
-import IconSend from "~/renderer/icons/Send";
+import IconQrCode from "~/renderer/icons/QrCode";
 import { hideEmptyTokenAccountsSelector } from "~/renderer/reducers/settings";
 import { openModal } from "~/renderer/actions/modals";
 import { setHideEmptyTokenAccounts } from "~/renderer/actions/settings";
@@ -71,7 +71,7 @@ const OptionsButton = () => {
     {
       key: "exportAccounts",
       label: t("accounts.optionsMenu.exportToMobile"),
-      icon: <IconSend size={16} />,
+      icon: <IconQrCode size={16} />,
       onClick: () => onOpenModal("MODAL_EXPORT_ACCOUNTS"),
     },
     {
diff --git a/src/renderer/screens/settings/sections/Accounts/index.js b/src/renderer/screens/settings/sections/Accounts/index.js
index c0da9f0e6..885e71451 100644
--- a/src/renderer/screens/settings/sections/Accounts/index.js
+++ b/src/renderer/screens/settings/sections/Accounts/index.js
@@ -3,7 +3,7 @@
 import React from "react";
 import { useTranslation } from "react-i18next";
 import TrackPage from "~/renderer/analytics/TrackPage";
-import EyeSlash from "~/renderer/icons/EyeSlash";
+import EyeOff from "~/renderer/icons/EyeOff";
 import { SettingsSection as Section, SettingsSectionHeader as Header } from "../../SettingsSection";
 import HideEmptyTokenAccountsToggle from "./HideEmptyTokenAccountsToggle";
 import SectionExport from "./Export";
@@ -18,7 +18,7 @@ const SectionAccounts = () => {
         <TrackPage category="Settings" name="Accounts" />

         <Header
-          icon={<EyeSlash />}
+          icon={<EyeOff />}
           title={t("settings.accounts.hideEmptyTokens.title")}
           desc={t("settings.accounts.hideEmptyTokens.desc")}
           renderRight={<HideEmptyTokenAccountsToggle />}
diff --git a/tests/specs/__image_snapshots__/firmware-update-0-manager-page-snap.png b/tests/specs/__image_snapshots__/firmware-update-0-manager-page-snap.png
index 935236aa6..9ba65e887 100644
Binary files a/tests/specs/__image_snapshots__/firmware-update-0-manager-page-snap.png and b/tests/specs/__image_snapshots__/firmware-update-0-manager-page-snap.png differ
diff --git a/tests/specs/__image_snapshots__/firmware-update-1-disclaimer-modal-snap.png b/tests/specs/__image_snapshots__/firmware-update-1-disclaimer-modal-snap.png
index 0dcae84fb..7ff8d2531 100644
Binary files a/tests/specs/__image_snapshots__/firmware-update-1-disclaimer-modal-snap.png and b/tests/specs/__image_snapshots__/firmware-update-1-disclaimer-modal-snap.png differ
diff --git a/tests/specs/__image_snapshots__/firmware-update-2-disclaimer-modal-checkbox-snap.png b/tests/specs/__image_snapshots__/firmware-update-2-disclaimer-modal-checkbox-snap.png
index e29af93b3..c568d1e4b 100644
Binary files a/tests/specs/__image_snapshots__/firmware-update-2-disclaimer-modal-checkbox-snap.png and b/tests/specs/__image_snapshots__/firmware-update-2-disclaimer-modal-checkbox-snap.png differ
diff --git a/tests/specs/__image_snapshots__/firmware-update-4-disclaimer-modal-continue-2-snap.png b/tests/specs/__image_snapshots__/firmware-update-4-disclaimer-modal-continue-2-snap.png
index a233281c0..47d793513 100644
Binary files a/tests/specs/__image_snapshots__/firmware-update-4-disclaimer-modal-continue-2-snap.png and b/tests/specs/__image_snapshots__/firmware-update-4-disclaimer-modal-continue-2-snap.png differ
diff --git a/tests/specs/__image_snapshots__/firmware-update-5-flash-mcu-start-snap.png b/tests/specs/__image_snapshots__/firmware-update-5-flash-mcu-start-snap.png
index 8b5dd7a1f..803b94923 100644
Binary files a/tests/specs/__image_snapshots__/firmware-update-5-flash-mcu-start-snap.png and b/tests/specs/__image_snapshots__/firmware-update-5-flash-mcu-start-snap.png differ
diff --git a/tests/specs/__image_snapshots__/firmware-update-6-flash-mcu-done-snap.png b/tests/specs/__image_snapshots__/firmware-update-6-flash-mcu-done-snap.png
index 516c4e454..2c4a1e5fd 100644
Binary files a/tests/specs/__image_snapshots__/firmware-update-6-flash-mcu-done-snap.png and b/tests/specs/__image_snapshots__/firmware-update-6-flash-mcu-done-snap.png differ
diff --git a/tests/specs/__image_snapshots__/firmware-update-7-close-modal-snap.png b/tests/specs/__image_snapshots__/firmware-update-7-close-modal-snap.png
index c9a811b94..1de61c821 100644
Binary files a/tests/specs/__image_snapshots__/firmware-update-7-close-modal-snap.png and b/tests/specs/__image_snapshots__/firmware-update-7-close-modal-snap.png differ
diff --git a/tests/specs/__image_snapshots__/firmware-update-8-back-to-dashboard-snap.png b/tests/specs/__image_snapshots__/firmware-update-8-back-to-dashboard-snap.png
index d0c663afd..fcead86cd 100644
Binary files a/tests/specs/__image_snapshots__/firmware-update-8-back-to-dashboard-snap.png and b/tests/specs/__image_snapshots__/firmware-update-8-back-to-dashboard-snap.png differ
```

**Comment by gre** [comment]
This PR was outdated and the branch is too old to be kept on **upstream** (it never should have been on upstream in first place)

I kept above a save of the diff that this PR was doing.

**Pull request closed without merging**